### PR TITLE
UI: update font family

### PR DESCRIPTION
**Pull request by Hypnosphi**
Based on [Slack discussion](https://storybooks.slack.com/archives/C4Q0S3MQR/p1501290120980016)

### CODE REVIEW

File: lib/ui/src/modules/ui/components/theme.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 export const baseFonts = {
   fontFamily: `
-    -apple-system, ".SFNSText-Regular", "San Francisco", "Roboto",
-    "Segoe UI", "Helvetica Neue", "Lucida Grande", sans-serif
+    -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu",
+    "Cantarell", "Fira Sans", "Droid Sans", "Arial", sans-serif
```
Comment (danielduan) [inline lib/ui/src/modules/ui/components/theme.js]: I think `Helvetica Neue` is still necessary for OS X 10.11 and below when not in Chrome:
https://bitsofco.de/the-new-system-font-stack/

Comment (Hypnosphi) [inline lib/ui/src/modules/ui/components/theme.js]: Shouldn't `Arial` be available there?

Comment (danielduan) [inline lib/ui/src/modules/ui/components/theme.js]: Yes it is. I think it's to conform to the system's original styles and Arial is an unfamiliar and clunky looking font to Mac users.

Comment (Hypnosphi) [inline lib/ui/src/modules/ui/components/theme.js]: Well if the alternative is to have a font that jumps when going from normal to bold, the unfamiliarity is a lesser evil

Comment (danielduan) [inline lib/ui/src/modules/ui/components/theme.js]: hm fair enough.

**Review by danielduan — APPROVED** [review]

**Comment by igor-dv** [comment]
When I [mentioned](https://github.com/storybooks/storybook/pull/1523#discussion_r130310345) changing the fonts is inconstant in the prev PR, I meant that IMO it's wrong to change this only in one place (lib/ui). This kind of theme is used in a few other places around this repo (for example addons), and styleguide-wise I think all the fonts should be the same (at least in the official addons / UI). But maybe I 'm too picky =) Anyway if it solves this jumping glitch -  :+1: :+1:

**Comment by Hypnosphi** [comment]
Well, in fact there isn't any consistensy of font-family across the codebase at the moment: https://github.com/storybooks/storybook/search?utf8=%E2%9C%93&q=fontFamily&type=

And if we want to make it more consistent, it would be better to use a shared theme instead repeating ourselves. This can be a part of the major task of moving UI components to `lib/components`

**Pull request merged**